pcdsdevices/lxe.py

```diff
@@ -482,6 +486,15 @@ def _offset_changed(self, value, **kwargs):
         """
         self.setpoint.user_offset = value
 
+    def check_hla(func):
+        def wrapper(self, *args, **kwargs):
```

Comment:
Consider decorating this with functools.wraps to keep the name/docstring of the original function
https://docs.python.org/3/library/functools.html#functools.wraps